content/help/ops/security/authorization-restrictive/index.md

```diff
@@ -18,7 +18,4 @@ matching requests should flow through. If all requests continue to be denied, yo
 `default` namespace (`metadata/namespace` line should be `default`). For non-Kubernetes environments, all `ServiceRoles` and `ServiceRoleBindings`
 for a mesh should be in the same namespace.
 
-1. Check Pilot and proxy log (or collect Pilot and proxy log and file a GitHub issue) to find out the exact cause. You could collect the LDS configuration generated by Pilot by running
-[`dump_kubernetes.sh`]({{< github_file >}}/tools/dump_kubernetes.sh).
-
-1. For Mixer based authorization, it is often the case that Istio Proxy to Mixer connection is down, Mixer attribute manifest is not setup correctly. Check Mixer Log (or collect Mixer log and file a GitHub issue) to find out the exact cause.
+1. Follow the [Debugging Authorization](/help/ops/security/debugging-authorization/) to find out the exact cause.
```

Comment:
Follow the \[Debugging Authorization docs\]\(/help/ops/security/debugging-authorization/\) to find out the exact cause.